### DISCUSSION TITLE
Welcome to Addonify Quick View Discussions Board!

### DISCUSSION THREAD

**Discussion by addonify** [addonify/addonify-quick-view#91] [Announcements]
## 👋 Welcome!

  We’re using Discussions as a place to connect with other members of our community. We hope that you:

  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome other members and be open-minded so we can build Adonify Quick View better together 💪.

## 🌜 Where to start?

Here are a few ways how you can engage with us. 

👉  Check the [ongoing discussion here](https://github.com/addonify/addonify-quick-view/discussions) or create a new one if required.

👉  Read the getting started [documentation guide here.](https://docs.addonify.com/kb/woocommerce-quick-view/)

👉  If you found a **BUG** you can [report that here.](https://github.com/addonify/addonify-quick-view/issues)

## ⚡️ Are you a Developer?

We would like to encourage great developers like you to review the source code & ask for commit access in the Discussion if you wish to improve Addonify Quick View.

**Comment by Chandrasekhar1211** [comment]
Hi, thanks for the plugin. I have selected all options in the dashboard, but the product title and rating is not coming inside the quick view modal.
![image](https://user-images.githubusercontent.com/106801870/233573501-89e3ec96-d8a0-4e73-b61f-ac6266511171.png)
![image](https://user-images.githubusercontent.com/106801870/233573689-c9487dff-4515-490c-be93-1a976fbca57d.png)
Can you please help me on this?

**Reply by h1dd3nsn1p3r** [reply]
Please share your website link.

**Reply by Chandrasekhar1211** [reply]
Hi... on home page, here you can find the quick view button. 
![image](https://user-images.githubusercontent.com/106801870/233581271-956cb93c-0794-498a-ac0d-d9aedf7f91e9.png)

**Reply by Chandrasekhar1211** [reply]
Hi, did you go through it?

**Reply by Chandrasekhar1211** [reply]
Hi @addonify, @h1dd3nsn1p3r ! let me know any updates on my query?

**Comment by Chandrasekhar1211** [comment]
Hi @addonify , @h1dd3nsn1p3r! Thank you for the plugin. The problem is we need the PDP design is different and the popup design to be different. But, for now, the content is calling using hooks from the PDP. So, whatever we update the style on the PDP, it is reflecting the same on the popup due to hooks. Let us know if can we customize the popup template using our custom PHP functions directly without hooks.

**Reply by Chandrasekhar1211** [reply]
Hi @addonify, @Asok17 let me know any updates on this please.

**Reply by Asok17** [reply]
Hi @Chandrasekhar1211 

Try hooking your custom functions into the action hook, `addonify_quick_view_product_summary`.

Before hooking your custom functions, unhook following functions, from the action:  

- `woocommerce_template_single_title`
- `woocommerce_template_single_rating`
- `woocommerce_template_single_price`
- `woocommerce_template_single_excerpt`
- `woocommerce_template_single_add_to_cart`
- `woocommerce_template_single_meta`

@see the function, `generate_contents()` in https://github.com/addonify/addonify-quick-view/blob/development/public/class-addonify-quick-view-public.php

I have not tried it. So I'm not sure if this works.

Let me know after you have tried it.

PS: If you are creating a new discussion, do not continue from here. Go to [Discussion page](https://github.com/addonify/addonify-quick-view/discussions) and click on `New discussion`.

**Comment by beatrice-scala** [comment]
Hi! I would like to know if the plugin is compatible with WooCommerce Composite Product and Variation Swatches for WooCommerce.

**Reply by Chandrasekhar1211** [reply]
Hi Team @Asok17 & @addonify, how are you? It's been a long time. We have updated our old plugin to the latest version. At that moment, the popup CSS style and functions got overwritten. You can see the difference. Can you help me towards the solution? Attaching screenshots for your reference below.

![image](https://github.com/user-attachments/assets/e8ac0abf-0ab0-4c95-8f84-b60113ad1cbe)
![image](https://github.com/user-attachments/assets/305cd656-a05c-465d-9478-c5f3c3addf8d)
**I think some classes were missing from the Version "1.2.7" to "1.2.17"**
